app/javascript/article-form/components/Close.jsx

```diff
@@ -1,7 +1,7 @@
 import { h } from 'preact';
 import { Button } from '@crayons';
 
-export const Close = () => {
+export const Close = ({ displayModal }) => {
```

Comment:
(nitpick / not blocking) - should we give this a default value? Just a suggestion because calling prop functions without default values makes me itchy, but in this case maybe it's fine since we only use it in one place and we know it's always passed in
```suggestion
export const Close = ({ displayModal = () => {} }) => {
```